content/docs/2.1/scalers/mongoDB.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "mongoDB"
+layout = "scaler"
+maintainer = "Community"
+description = "Scale applications based on mongoDB."
+availability = "v2.1+"
+go_file = "mongoDB_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the mongoDB trigger that scales based on result of mongoDB query.
```

Comment:
A big nit but can you replace all `mongoDB` with `MongoDB` please? (except for trigger type references of course.